api/tests/integration/infrastructure/repositories/user-orga-settings-repository_test.js

```diff
@@ -64,4 +64,34 @@ describe('Integration | Repository | UserOrgaSettings', function() {
     });
   });
 
+  describe('#update', () => {
+
+    let newOrganizationId;
+
+    beforeEach(async () => {
+      databaseBuilder.factory.buildUserOrgaSettings({ userId, currentOrganizationId: organizationId });
+      newOrganizationId = databaseBuilder.factory.buildOrganization().id;
+      await databaseBuilder.commit();
+    });
+
+    it('should return the current organizationId', async () => {
```

Comment:
`it('should return the updated organizationId')`

Reply:
Done